wagtail/core/blocks/stream_block.py

```diff
@@ -62,6 +63,15 @@ def sorted_child_blocks(self):
         return sorted(self.child_blocks.values(),
                       key=lambda child_block: child_block.meta.group)
 
+    def grouped_child_blocks(self):
+        """
+        Child blocks organised into groups, returned as an iterable of
+        (group_name, list_of_blocks) tuples
```

Comment:
I can update the comment to clarify this, but on the Python code side I think we're stuck with the terminology weirdness (instances of `Block` being block definitions rather than actual blocks of content, which are `BoundBlock`) due to the fact that writing a StreamField block definition involves creating instances of CharBlock etc. (Further rambling to that effect here... https://github.com/wagtail/wagtail/pull/5976#issuecomment-708620105)